libbfddp/bfddp_packet.h

```diff
@@ -33,6 +33,41 @@
 
 #include <stdint.h>
 
+/*
+ * Protocol definitions.
+ */
+
+/**
+ * BFD protocol version as defined in RFC5880 Section 4.1 Generic BFD Control
+ * Packet Format.
+ */
+#define BFD_PROTOCOL_VERSION 1
+
+/** Default data plane port. */
+#define BFD_DATA_PLANE_DEFAULT_PORT 50700
+
+/** BFD single hop UDP port, as defined in RFC 5881 Section 4. Encapsulation. */
+#define BFD_SINGLE_HOP_PORT 3784
+
```

Comment:
Consider adding `#define BFD_SINGLE_HOP_ECHO_PORT 3785` or something like that just to be complete.